apps/server/cmd/server/main.go

```
@@ -59,10 +67,24 @@ func controlPlaneHandler(ctx context.Context, databaseURL string) (http.Handler,
+	}
 	return &applicationHandler{
 		Handler:  httpapi.NewRouter(services.ControlPlane),
 		services: services,
-	}, database.Close, nil
+	}, closeApplication, nil
+}
+
+func reconcileRuntimeInstances(ctx context.Context, handler http.Handler) error {
+	application, ok := handler.(*applicationHandler)
+	if !ok || application.services == nil || application.services.RuntimeInstances == nil {
+		return fmt.Errorf("Runtime Instance service is unavailable")
```

golangci-lint (2.13.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)